js/xt.js

```diff
@@ -0,0 +1,1072 @@
+const Exchange = require ('./base/Exchange');
+const { TICK_SIZE } = require ('./base/functions/number');
+const Precise = require ('./base/Precise');
+const { ExchangeError } = require ('./base/errors');
+// ^^^ BadSymbol, BadRequest, OnMaintenance, InvalidOrder, ArgumentsRequired AccountSuspended, PermissionDenied, RateLimitExceeded, ExchangeNotAvailable, OrderNotFound, InsufficientFunds, AuthenticationError
+
+module.exports = class xt extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'xt',
+            'name': 'XT',
+            'countries': [ 'SC' ], // Seychelles
+            // 10 requests per second => 1000ms / 10 = 100 (All other)
+            // 3 requests per second => 1000ms / 3 = 333.333 (get assets -> fetchMarkets & fetchCurrencies)
+            // 1000 times per minute for each single IP -> Otherwise account locked for 10min
+
```

Comment:
blank lines inside objects are not allowed, `eslint` should have warned you, do you have it properly set up?